.github/workflows/build-reuse-win.yml

```diff
@@ -87,6 +87,12 @@ jobs:
       if: inputs.test == ''
       shell: pwsh
       run: scripts/build.ps1 -Config ${{ inputs.config }} -Platform ${{ inputs.plat }} -Arch ${{ inputs.arch }} -Tls ${{ inputs.tls }} ${{ inputs.xdp }} ${{ inputs.sanitize }} ${{ inputs.static }}
+    - name: Filter Build Artifacts
+      shell: pwsh
+      run: |
+        Remove-Item artifacts/bin/xdp -Recurse -Force -ErrorAction Ignore
+        Remove-Item artifacts/corenet-ci-main -Recurse -Force -ErrorAction Ignore
+        Remove-Item artifacts/xdp -Recurse -Force -ErrorAction Ignore
```

Comment:
Are you seeing an issue with these files? I remember it's sort of intentional, so the test workflow won't need to download them again.

Reply:
I'm trying to reduce the size to optimize download/copy times. For one, there are two copies of the XDP binaries. Do we really need both? Are you sure the other stuff is used?

Reply:
I don't remember why we have 2 copies of xdp. corenet-ci-main is uploaded so the tests for duonic/kernel do not have to download stuff externally.

How much space have you saved though? I remember I tried to trim the artifacts and I then realized the gain was only like tens of MBs or even less.

Reply:
We probably don't need xdp binaries in tests anymore as we now install the msi when preparing the environment?

Reply:
Here's a before and after comparison:
```
Release-windows-windows-2022-x64-schannel     204 MB	
Release-winkernel-windows-2022-x64-schannel   104 MB

Release-windows-windows-2022-x64-schannel     121 MB
Release-winkernel-windows-2022-x64-schannel    21 MB
```

Reply:
So far, tests seem to be passing, so I don't think we need any of the other stuff.

Reply:
They are for speeding up the tests and reducing flakiness caused by downloading extra stuff because prepare-machine.ps1 will download them if not present.

I will leave it up to you. Not a big deal to me.

Reply:
(Removing them saves a lot of space for release schannel but not so much for openssl)